api/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Should probably add this to `.gitignore`?

Reply:
I thought it might be useful for others, as long as people are using vscode, of course.
Just an example of debugging launch script. Happy to exclude though if you think its not useful.